Percy.Test/Percy.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net47</TargetFrameworks>
+    <TargetFrameworks>net7.0</TargetFrameworks>
```

Comment:
Why are we changing this?

Reply:
This is needed as we are adding packages for unit test which need dotnet 7. also FYI appium SDK is also using 7 https://github.com/percy/percy-appium-dotnet/blob/main/Percy.Tests/Percy.Tests.csproj#L3

Reply:
Cool, .NET Target for Test project shouldn't affect SDK